src/styles/blocks/main-navigation.scss

```diff
@@ -0,0 +1,75 @@
+@import "../utils/variables";
+
+.main-navigation {
+  margin-right: 50px;
+  overflow-x: scroll;
+  overflow-y: hidden;
+
+  &__list {
+    height: 60px;
+    display: flex;
+    margin: 0;
+    line-height: 60px;
+    padding: 0;
+  }
+  &__item {
+    list-style: none;
+    text-transform: uppercase;
+    height: 60px;
+    display: flex;
+    flex-direction: column;
+    color: $main-accent;
+  }
+
+  &__item:not(:first-of-type) {
+    margin-left: 20px;
+  }
+
+  &__item:first-of-type::after {
+    position: relative;
+    bottom: 4px;
+    content: "";
+    background-color: #00acdc;
+    height: 0;
+    padding: 2px 0;
+    border-radius: 2px;
+    width: 100%;
+  }
+
+  &__item > a {
+    text-decoration: none;
+    color: $main-accent;
+    transition-duration: 0.3s;
+  }
+
+  &__item > a:hover {
+    color: $blue-accent;
+  }
+  .is-active {
+    color: $blue-accent;
+  }
+}
+
+.logo {
+  align-self: center;
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
```

Comment:
if logo is a separate block so you should move it to separate file, but to my mind it's better to change a little bit your header and use other class names - example 'header__logo' and 'header__navigation'